libkbfs/interfaces.go

```diff
@@ -107,6 +107,12 @@ type KBFSOps interface {
 	// isn't favorited.
 	DeleteFavorite(ctx context.Context, fav Favorite) error
 
+	// GetOrInitializeNewMDMaster gets the existing MD or initialize a new MD for
+	// master branch
+	GetOrInitializeNewMDMaster(ctx context.Context, mdops MDOps, h *TlfHandle) (
```

Comment:
Actually a third option would be making a method on the `KBFSOps` interface that just returns the keys as byte slices.  That might be useful outside this library.